tests/test-edit-flow-notifications.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+class WP_Test_Edit_Flow_Notifications extends WP_UnitTestCase {
+
+	protected static $admin_user_id;
+	protected static $EF_Notifications;
```

Comment:
Minor: should probably be lower-case?

```suggestion
	protected static $ef_notifications;
```

And updated throughout the class.